options.js

```diff
@@ -1,15 +1,21 @@
-const eslint = require('eslint')
-const path = require('path')
-const pkg = require('./package.json')
+import { dirname, join } from 'node:path'
+import { readFileSync } from 'node:fs'
+import { fileURLToPath } from 'node:url'
+import eslint from 'eslint'
 
-module.exports = {
-  bugs: pkg.bugs.url,
+const __dirname = dirname(fileURLToPath(import.meta.url))
+const json = readFileSync('./package.json', 'utf8')
+const pkg = JSON.parse(json)
+const { bugs, homepage, version } = pkg
```

Comment:
I personally prefer these to be referred to as e.g. `pkg.verison` below, one less level of indirection to scan when reading the code...

Reply:
I also don't like to destruct everything. I agree with you in some scenarios